### PR TITLE
minor UI tweaks in mapping tab. 

### LINKED ISSUE #608: De-select "show data preview" on Mapping tab by default.

### PR DESCRIPTION
**Pull request by jwildfire**
# Overview

Minor updates in mapping tab:
- Deselect the data preview checkbox by default
- Add title and description
- Fix minor bug that stopped data standard in mapping header for each domain from renderering #527 

fixes #608

# Test notes

Run the app and make sure the mapping tab updates are looking good.

### CODE REVIEW

File: R/mod_mappingTab.R [hunk 1]
```diff
@@ -31,8 +31,10 @@ mappingTabUI <- function(id, meta, domainData, mappings=NULL, standards=NULL){
   metaDomains <- unique(meta$domain)
   dataDomains <- names(domainData)
   domains <- metaDomains[metaDomains %in% dataDomains]
-
+  print(standards)
```
Comment (elimillera) [inline R/mod_mappingTab.R:34]: Same comment here. Should this be removed?

**Review by elimillera — APPROVED** [review]
Tested it out and it looks good. Couple of debug statements might have been left in.

File: R/mod_mappingTab.R (hunk 1 above)
Comment (jwildfire) [reply R/mod_mappingTab.R:34]:
```suggestion

```

File: R/app_startup.R [hunk 2]
```diff
@@ -69,12 +69,13 @@ app_startup<-function(domainData=NULL, meta=NULL, charts=NULL, mapping=NULL, aut
     # generate mappings and data standards
     mappingObj <- makeMapping(domainData, meta, autoMapping, mapping)
 
+    print(mappingObj)
```
[the remark this answers is not in the document]
Comment (jwildfire) [reply R/app_startup.R:72]:
```suggestion

```

**Comment by jwildfire** [comment]
> Couple of debug statements might have been left in.

Thanks - they were definitely strays. Deleted now.

**Pull request merged**